index.html

```diff
@@ -24,7 +9,8 @@
     <meta property="og:description" content="A community of software engineers and product developers, originally created for students of KubSAU">
     <meta property="og:image" content="https://github.com/move-fast-and-break-things/mfabt_site/assets/92587254/18e628c2-26bb-4f31-8a7f-5173e218ed40">
     <meta property="og:url" content="https://movefastandbreakthings.club/">
-    <link rel="stylesheet" href="main.css">
+    <link rel="preload" href="main.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
+    <noscript><link rel="stylesheet" href="main.css"></noscript>
```

Comment:
Does this have any impact on our case?

If not, let's revert it to the original style declaration for simplicity 😄

Reply:
initially i thought it was needed to increase lighthouse score on performance aspect, but when i tested it again it doesn't really increase the score. I reverted it to the original style